kola/tests/kubeadm/kubeadm.go

```diff
@@ -75,6 +75,38 @@ var (
 	// testConfig holds params for various kubernetes releases
 	// and the nested params are used to render script templates
 	testConfig = map[string]map[string]interface{}{
+		"v1.26.0": map[string]interface{}{
+			"MinMajorVersion": 3033,
+			// from https://github.com/flannel-io/flannel/releases
+			"FlannelVersion": "v0.19.1",
```

Comment:
I don't remember if we agreed on any policy with regards to bumping these version numbers. Have we? I'm asking because looking into the URLs in comments, I can see that flannel has version v0.20.2, cilium has 1.12.5, cilium cli has 0.12.12, cni is still v1.1.1, crictl has v1.26.0 and k8s release is still v0.14.0.

So not sure, maybe a follow-up PR if these versions work for k8s 1.26?

(Also, happy that the checksums script worked for you!)

Reply:
My take on these tests is that we want to test a range of k8s versions on flatcar. Since k8s needs some networking solution and these tend to be quite dependent on running kernel, we also want to test a range of networking solutions versions. So it may be a good idea to bump the versions of all this stuff at the time of a new major k8s release. It won't give 100% coverage, but should be good enough. This is in light of a realization that having data on the most used version combinations of software is a wishful thinking, not gonna happen. :)